cmd/microservice-solana-worker-server/main.go

```diff
@@ -186,13 +187,21 @@ func main() {
 				)
 			}
 
+			pools := []worker_types.PoolDetails{
```

Comment:
Worth noting that we mock this internally since our Solana infra is not as mature (owing to the Solend situation)